share/doc/homebrew/Formula-Cookbook.md

```diff
@@ -303,14 +303,15 @@ correct. Add an explicit [`version`](http://www.rubydoc.info/github/Homebrew/bre
 
 ## Commit
 
-Everything is built on Git, so contribution is easy:
-
+Everything is built on Git, so contribution is easy:  
+1. Fork the Homebrew/homebrew-core repository on GitHub. This creates a pushable, personal remote repository. This is needed as only Homebrew maintainers have push access to the main repository.  
+2. Add the pushable forked repository with `git remote add YOUR_USERNAME https://github.com/YOUR_USERNAME/homebrew-core.git`
```

Comment:
Let's put `YOUR_USERNAME` in brackets as `<YOUR_GITHUB_USERNAME>` so it's clear that it's a placeholder for replacement text, and the GitHub username (not the local machine username).